wsd/DocumentBroker.cpp

```diff
@@ -4030,26 +4030,34 @@ void DocumentBroker::syncBrowserSettings(const std::string& userId, const std::s
                                                  << "] for all sessions with userId [" << userId
                                                  << ']');
 
-    bool upload = false;
+    // update browsersetting for all the session
+    std::shared_ptr<ClientSession> sessionForUpload = nullptr;
```

Comment:
This initialization is not necessary, and it's actually better not to do what by default is already done. The following is sufficient, efficient, and clear.
```
std::shared_ptr<ClientSession> sessionForUpload;
```